pom.xml

```diff
@@ -1102,11 +1105,7 @@
         <artifactId>license-maven-plugin</artifactId>
         <version>2.8</version>
         <configuration>
-          <header>com/mycila/maven/plugin/license/templates/APACHE-2.txt</header>
-          <properties>
-            <owner>LinkedIn Corp.</owner>
-            <email>pinot-core@linkedin.com</email>
-          </properties>
+          <header>license.header</header>
           <excludes>
```

Comment:
Please check all the excludes as well. Some of them might not make sense, some of them might need to be updated.